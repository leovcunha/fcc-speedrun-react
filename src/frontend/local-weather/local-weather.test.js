import React from "react";
import renderer from "react-test-renderer";
import LocalWeather from "../local-weather";

describe("tribute-page", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<LocalWeather/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
