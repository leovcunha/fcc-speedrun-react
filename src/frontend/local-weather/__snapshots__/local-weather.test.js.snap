// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tribute-page renders without crashing 1`] = `
<div
  className="local-weather jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Local Weather
  </div>
  <hr
    className="p-5"
  />
  <div
    className="row justify-content-center h4"
  >
    Loading
  </div>
  <div
    className="row justify-content-center"
  />
  <div
    className="row justify-content-center h5"
  >
    
  </div>
  <div>
    <div
      className="row justify-content-center h5"
    >
       Temperature is:  
      <h5>
        -273.0
         C
      </h5>
    </div>
    <div
      className="row justify-content-center h5"
    >
       Max:  
      <h5>
        -273.0
         C
      </h5>
    </div>
    <div
      className="row justify-content-center h5"
    >
       Min:  
      <h5>
        -273.0
         C
      </h5>
    </div>
  </div>
  <div
    className="row justify-content-center mt-5"
  >
    <button
      className="btn btn-secondary"
      onClick={[Function]}
      type="button"
    >
      Change Temperature Scale
    </button>
  </div>
</div>
`;
